src/test/java/com/google/javascript/gents/multiTests/import_rewrite/export.ts

```diff
@@ -1,23 +1,20 @@
 export function B() {}
 
-export const x = 4;
-
-export const y = 8;
+export const C = {
+  x: 4,
+  y: 8
+};
 
 export function D() {}
-
-export function foo() {}
+D.foo = function() {};
 
 export function E() {}
-
-export function bar() {}
+E.bar = function() {};
```

Comment:
Haha. I'll file an issue to set up a test that compiles the `.ts` code, we should find these in a structured way.